SIT.Manager.Avalonia/Views/MainWindow.axaml

```diff
@@ -13,8 +13,7 @@
 		Width="800"
 		Height="475"
 		WindowStartupLocation="CenterScreen"
-		TransparencyLevelHint="AcrylicBlur"
-		Background="Transparent"
+		Background="Black"
```

Comment:
Not sure why this is here. It's possible that these options aren't needed but I'm not a fan of changing things unrelated to the PR and undocumented

Reply:
I think it's better to remove AcrylicBlur because we literally do not use transparent AcrylicBlur background, it can make a difference in perfomance when you render a window.

Reply:
Can we move this to another PR. I don't disagree that these are possibly useless but it's outside the scope of this PR and I don't like making undocumented changes